src/giskardpy/plugin_tf_publisher.py

```diff
@@ -0,0 +1,62 @@
+import rospy
+import numpy as np
+from geometry_msgs.msg import TransformStamped
+from py_trees import Status
+from tf2_msgs.msg import TFMessage
+
+import giskardpy.identifier as identifier
+from giskardpy import logging
+from giskardpy.plugin import GiskardBehavior
+from giskardpy.utils import normalize_quaternion_msg
+
+
+class TFPlugin(GiskardBehavior):
+    """
+    TODO
+    """
+
+    def __init__(self, name):
+        super(TFPlugin, self).__init__(name)
+        self.original_links = set(self.get_robot().get_link_names())
+        self.tf_pub = rospy.Publisher(u'/tf', TFMessage, queue_size=10)
```

Comment:
while we are at it, might as well make `/tf` a parameter you can set in the config file